documentation/modules/proc-enabling-tls.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-enabling-tls-{context}']
+= Accessing the cluster
+
+As `loadbalancer` is used for external access to the cluster, a cluster CA certificate is required to enable TLS (Transport Layer Security) encryption between the broker and the client.
+
+.Prerequisites
+
+* A Kafka cluster running within the Kubernetes cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Find the IP address of the bootstrap `loadbalancer`:
++
+[source, shell, subs=+quotes]
+----
+kubectl get service my-cluster-kafka-external-bootstrap -o=jsonpath='{.status.loadBalancer.ingress[0].ip}{"\n"}'
+----
++
+Use the IP address together with port 9094 in your Kafka client as the bootstrap address.
```

Comment:
As the bootstrap address _for what_? Your Kafka producer or consumer, presumably?